index.html

```diff
@@ -0,0 +1,206 @@
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="flex project_coding_langs">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button class="preview_button outlinings">See Project</button>
+        </article>
+        <article class="card flex white-back">
+          <img src="./images/project_3.png" alt="tonic project preview" />
+          <h3>Tonic</h3>
+          <ul class="flex project_details">
+            <li>CANOPY</li>
+            <li>BACKEND DEV</li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="flex project_coding_langs">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button class="preview_button outlinings">See Project</button>
+        </article>
+        <article class="card flex white-back">
+          <img src="./images/project_4.png" alt="Multi-Post project preview" />
+          <h3>Multi-Post Stories</h3>
+          <ul class="flex project_details">
+            <li>CANOPY</li>
+            <li>BACKEND DEV</li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="flex project_coding_langs">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button class="preview_button outlinings">See Project</button>
+        </article>
+      </section>
+      <section class="about white-back">
+        <div>
+          <h1>
+            About <br />
+            Myself
+          </h1>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <div class="socials">
+            <h6>LET'S CONNECT</h6>
+            <ul class="icons_social__container flex">
+              <li>
+                <a href="#"><i class="fa-brands fa-twitter"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa-brands fa-linkedin-in"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa-brands fa-github"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa-brands fa-instagram"></i></a>
+              </li>
+            </ul>
+          </div>
```

Comment:
You have done good work following Figma Design. The `font-size `of the icons as per Figma design is almost `20px`. The font-size you are using is 30px.

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/219709459-13145b2a-e568-4ceb-93eb-05c46d7fc2ac.png)  | ![image](https://user-images.githubusercontent.com/61361037/219709373-b6e3d3b4-5970-4f4f-a1f8-469e43b55920.png)  |